### ISSUE TITLE
Is there anyway to modify G/D ratio for pix2pix model?

### ISSUE THREAD

**Issue by hyecheol123** [junyanz/pytorch-CycleGAN-and-pix2pix#1097]
Though I adjusted number of layers and the number of convolution filters in Discriminator, I figured out that the Discriminator is still too strong. In order to further depower Discriminator, I want to modify G/D ratio, but I cannot find how to do so.
Are there any specific points that those information coded?

**Comment by junyanz** [comment]
You can also change learning rates. See this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/models/pix2pix_model.py#L69).

**Comment by hyecheol123** [comment]
Just use opt.lr * 0.5, for instance?

**Comment by hyecheol123** [comment]
May I ask you how can I calculate the G/D based on the modified learning rate? If it is G:D = 1:0.5 = 2:1, would it be okay to understand that the original model's G/D ratio is 1?

**Comment by junyanz** [comment]
The original ratio is 1. You can fix one learning rate and change the other one.

**Comment by hyecheol123** [comment]
Thanks!!
